src/pages/FavoritesPage/FavoritesPage.tsx

```
@@ -0,0 +1,99 @@
+            <p className="text-body text-muted-foreground">
+              Products you've saved for later
+            </p>
+          </div>
+        </div>
+
+        {/* Category Filter */}
+        <div>
+          <label className="text-bodySmall mb-2 flex items-center gap-2 font-medium">
+            <Filter className="h-4 w-4" />
+            Filter by Category
+          </label>
```

Biome (2.3.13)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)